### DISCUSSION TITLE
Can't SSH to host from inside OliveTin container

### DISCUSSION THREAD

**Discussion by TheBig-O** [OliveTin/OliveTin#410] [Q&A]
I have been trying to figure out why I can't SSH from inside OliveTin for days. I have tested the SSH keys from other machines  (non-docker) and they work perfectly. when I try to ssh out using the OliveTin container, I get `Permission denied (publickey)`.
From the web interface, my errors are:
```
exit status 255

root@192.168.1.###: Permission denied (publickey).

OliveTin::shellAfterCompleted stdout
OliveTin::shellAfterCompleted stderr
OliveTin::shellAfterCompleted errors and summary
Your shellAfterCompleted exited with code 0
OliveTin::shellAfterCompleted output complete
```

Anyone have any ideas for how I can make this work? Without SSH, I can't do many of the scripts I'd like to use.
thanks for the help.

**Comment by LeonNamowitz** [comment]
Just stumbled upon the same issue. The way I managed to make it work is by specifying the rsa key with the "ssh -i" flag. 
Presuming you mounted the key based on the documentation: 

`shell: ssh -i /home/olivetin/.ssh/id_rsa root@192.168.178.21 "/my_server/my_example_script.sh"`
